smart-py/objkt_swap_tests.py

```diff
@@ -0,0 +1,852 @@
+import smartpy as sp
+import os
+
+OBJKTSwap = sp.io.import_script_from_url(f"file://{os.getcwd()}/objkt_swap.py").OBJKTSwap
+
+@sp.add_test("Class constructs with default values")
+def test():
+
+    # init test and create html output
+    scenario = sp.test_scenario()
+    scenario.h1("Swap Objkt Test")
+
+    # init test values
+    seller = sp.test_account("seller")
+    manager = sp.test_account("manager")
+    curator = sp.test_account("curator")
+    objkt = sp.test_account("objkt123")
+    hdao = sp.test_account("hdao")
+
+    creator = sp.test_account("creator")
+    metadata = sp.record(
```

Comment:
The same comment as the other test file for making this a reusable utility call for setting the metadata for each test.

Reply:
great idea to start getting the tests smaller